pkg/zfs/zfs_util.go

```diff
@@ -115,6 +116,30 @@ func buildZvolCreateArgs(vol *apis.ZFSVolume) []string {
 	return ZFSVolArg
 }
 
+// builldZFSSnapCreateArgs returns zfs create command for zfs snapshot
```

Comment:
Examples of command as a comment can help

Reply:
sure.